app/src/main/cpp/ControllerContainer.cpp

```diff
@@ -74,8 +74,8 @@ ControllerContainer::InitializePointer() {
   }
   CreationContextPtr create = m.context.lock();
   VertexArrayPtr array = VertexArray::Create(create);
-  const float kLength = -5.0f;
-  const float kHeight = 0.0008f;
+  const float kLength = -1.0f;
+  const float kHeight = 0.005f;
 
   array->AppendVertex(Vector(-kHeight, -kHeight, 0.0f)); // Bottom left
   array->AppendVertex(Vector(kHeight, -kHeight, 0.0f)); // Bottom right
```

Comment:
I think the indices are wrong from when the pointer was first created. I'll play around with them and see I can get the triangle surface normals going the correct direction,